tests/test_config.yaml

```diff
@@ -1,4 +1,4 @@
-output path: "tests/"
+output path: "tests/transformed"
```

Comment:
This should be inside resources too.

Reply:
And I suggest making output path `tests/resources/actual`. So, accidentally running cpplint wouldn't get errors.